pages/news.html

```diff
@@ -88,6 +88,7 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
+	    <p><strong>27/09/2023</strong> - Uyuni 2023.09 is now available! The OS for the Server and Proxies is now openSUSE Leap 15.5, so major upgrade procedures will be needed! Salt is now updated to 3006.0 and it will require a special update procedure. Check the <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details and all the other new features</p>
```

Comment:
```suggestion
	    <p><strong>28/09/2023</strong> - Uyuni 2023.09 is now available! The OS for the Server and Proxies is now openSUSE Leap 15.5, so major upgrade procedures will be needed! Salt is now updated to 3006.0 and it will require a special update procedure. Check the <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details and all the other new features</p>
```